com.unity.ml-agents/Runtime/DiscreteActionMasker.cs

```diff
@@ -15,19 +14,13 @@ namespace Unity.MLAgents
     /// may be illegal. For example, if an agent is adjacent to a wall or other obstacle
     /// you could mask any actions that direct the agent to move into the blocked space.
     /// </remarks>
-    public class DiscreteActionMasker
+    public class DiscreteActionMasker : IDiscreteActionMask
```

Comment:
Just checking - this isn't considered a breaking change, right? I've seen some chatter about things the look harmless because the old code still compiles, but the assemblies would be incompatible.

Reply:
good question.  I wonder if we can check the version bump with the validation suite somehow

Reply:
My intuition would be that since the default behavior didn't change, and the method's signature that uses this type is concretely defined, therefore, it should be ok.  Would be good to know the answer to this question concretely.